packages/markup-include/src/CdcMarkupInclude.tsx

```diff
@@ -73,8 +80,13 @@ const CdcMarkupInclude: React.FC<CdcMarkupIncludeProps> = ({ configUrl, config:
 
     response.data = responseData
     const processedConfig = { ...(await coveUpdateWorker(response)) }
+    const configNeedsUpdate = !(
```

Comment:
I feel like you need to handle this in the useEffect and not in the function itself.